contracts/ERC677MultiBridgeToken.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity 0.4.24;
+
+import "./ERC677BridgeToken.sol";
+
+/**
+ * @title ERC677MultiBridgeToken
+ * @dev This contract extends ERC677BridgeToken to support several bridge simulteniously
+ */
+contract ERC677MultiBridgeToken is ERC677BridgeToken {
```

Comment:
Consider to inherit this new contract by `ERC677BridgeTokenRewardable`. Please elaborate which components could be also affected after such kind of change.